File: UserManagement.API/UserManagment.Service/Models/UserModel.cs
using System.ComponentModel.DataAnnotations;

namespace UserManagment.Service.Models
{
    public class UserModel
    {
        public string? Id { get; set; }
        public string? Username { get; set; }
        public string? Name { get; set; }
        public string? Email { get; set; }
        public string Role { get; set; }

    }
}

File: UserManagement.Client/src/app/models/user.model.ts
export default class UserModel{
    name:string;
    username:string;
    email:string;
    role:string;
    actionUser:string;

    constructor(name:string,username: string,email:string,role:string,actionUser:string) {
        this.name = name;
        this.username = username;
        this.email = email;
        this.role = role;
        this.actionUser = actionUser;
    }
}